relay-sampling/src/config.rs

```diff
@@ -140,18 +164,16 @@ pub enum SamplingValue {
     /// until a sample rate rule is found. The matched rule's factor will be multiplied with the
     /// accumulated factors before moving onto the next possible match.
     Factor {
-        /// The fator to apply on another matched sample rate.
+        /// The factor to apply on another matched sample rate.
         value: f64,
     },
-}
-
-impl SamplingValue {
-    pub(crate) fn value(&self) -> f64 {
```

Comment:
function no longer makes sense because reservoir limit doesn't have an analogous value to these